lib/ide/src/view/project_view.rs

```diff
@@ -0,0 +1,87 @@
+//! This module contains ProjectView, the main view, responsible for managing TextEditor and
```

Comment:
"Project" means a set of Enso source files on disk. "project" is like rust "crate". We should be careful about names here. If I understand correctly, this is the "application" view here, right? Or maybe "root" view?

Reply:
But we will in some time allow to read different projects, right? So view displaying one project is called `ProjectView`